ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -142,7 +142,7 @@
       class="item indicator"
       data-cy="settings"
       class:active={path.active(path.settings(), $location)}>
-      <Tooltip value="Settings">
+      <Tooltip value="Settings" style="width: 100%; height: 100%">
```

Comment:
Wouldn't be simpler to wrap the `li` in the tooltip?

Reply:
That would not be valid HTML since only `li` elements can be nested inside `ul` elements